src/recommendations/install.md

```diff
@@ -0,0 +1,45 @@
+---
+group: product-recommendations
+title: Product Recommendations
+---
+
+Product recommendations are a powerful marketing tool you can use to increase conversions, boost revenue, and stimulate shopper engagement. Product recommendations are surfaced on the storefront in the form of units such as “Customers who viewed this product also viewed”. Because these suggestions are backed by a deep analysis of aggregated visitor data, they result in highly engaging, relevant, and personalized experiences for the shopper.
+
+Using recommendations on your storefront requires:
+
+1. Behavioral data - Needed to compute the product affinities used as the basis for Recommendations
```

Comment:
```suggestion
1. Behavioral data—Needed to compute the product affinities used as the basis for Recommendations
```

Reply:
looks like you're capitalizing "Recommendations" as a proper noun in this list, but not in the preceding paragraph. Should be consistent.